elastic-agent-shipper.spec.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+outputs:
+  - name: elasticsearch
+    description: Elasticsearch
+    platforms: &platforms
+      - linux/amd64
+      - linux/arm64
+      - darwin/amd64
+      - darwin/arm64
+      - windows/amd64
+      - container/amd64
+      - container/arm64
+    command: {}
+  - name: logstash
+    description: Logstash
+    platforms: *platforms
+    command: {}
```

Comment:
```suggestion
```

We should avoid including this until we actually implement a Logstash output: https://github.com/elastic/elastic-agent-shipper/issues/129

The Elasticsearch output is already in progress at least, so we can keep that one.

Reply:
makes sense, thanks